### ISSUE TITLE
failing install via composer

### ISSUE THREAD

**Issue by nuryagdym** [opencart/opencart#8186]
Hi,
I am trying to install opencart via composer by running:
`composer create-project opencart/opencart`
installation finished successfully, however, I first time try open page in browser it redirects me to 
/install/index.php
where I am getting the following errors: 
`Warning: require(/home/vagrant/nnkaravan/upload/system/storage/vendor/composer/../react/promise/src/functions_include.php): failed to open stream: No such file or directory in /home/vagrant/nnkaravan/upload/system/storage/vendor/composer/autoload_real.php on line 66`

` Fatal error: require(): Failed opening required '/home/vagrant/nnkaravan/upload/system/storage/vendor/composer/../react/promise/src/functions_include.php' (include_path='.:/usr/share/php') in /home/vagrant/nnkaravan/upload/system/storage/vendor/composer/autoload_real.php on line 66`

Also, I tried to set up opencart by running this command after creating project via composer:
`php upload/install/cli_install.php install --db_hostname localhost --db_username homestead --db_password secret --db_database nnkaravan --username admin --password password --email nuryagdy@rentrent.com.tr --http_server http://nnkaravan.dev/`

which also throws error:
```
PHP Warning:  Use of undefined constant DIR_STORAGE - assumed 'DIR_STORAGE' (this will throw an Error in a future version of PHP) in /home/vagrant/nnkaravan/upload/system/startup.php on line 70

Warning: Use of undefined constant DIR_STORAGE - assumed 'DIR_STORAGE' (this will throw an Error in a future version of PHP) in /home/vagrant/nnkaravan/upload/system/startup.php on line 70
SUCCESS! Opencart successfully installed on your server
Store link: http://nnkaravan.dev/
Admin link: http://nnkaravan.dev/admin/
```

**Server / Test environment (please complete the following information):**
 - Vagrant
 - Windows
 - PHP version 7.4.4

**Comment by nuryagdym** [comment]
asked in [community ](https://forum.opencart.com/viewtopic.php?f=199&t=219456&p=798079#p798079) and got an for error
`No such file or directory in`:
deleting `upload/system/storage/vendor` and running `composer update` solved issue.

error with `cli_install.php` still exist

**Comment by prabhatkumaroc082** [comment]
fix #9064
